spec/models/manageiq/providers/openshift/container_manager/refresh_parser_spec.rb

```diff
@@ -166,6 +166,32 @@
         parser.instance_variable_get('@data_index')[:container_image][:by_ref_and_registry_host_port].values[0])
       expect(parser.instance_variable_get('@data')[:container_images][0][:architecture]).to eq('amd64')
     end
+
+    context "image registries from openshift images" do
+      after :each do
+        inventory = {"image" => [image_from_openshift,]}
```

Comment:
Is that trailing comma expected?  It doesn't do anything, so I think it should be removed